index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <title>Bounce</title>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+</head>
+<body>
+    <div>
+        <h1 id="title">Bounce</h1>
+        <p id="description">
+            If you want to relax and free your mind for a couple of minutes just stay here and watch the balls bounce away.
+        </p>
+        <div id="ballCountDiv">
+            <label id="ballCountLabel">How many balls? </label>
+            <input type="number" id="ballCountInput">
+        </div>
+        <button id="startButton">Bounce</button>
+        
+    </div>
+
+    <div id="bounceDiv" ></div>
+    
+</body>
+<script src='main.js'></script>
```

Comment:
Why is this outside of the body tag?

Reply:
I moved the script tag inside the body tag.